cdk/apps/ica_credentials/cdk/app.py

```diff
@@ -72,25 +78,7 @@
     SLACK_WEBHOOK_SSM_NAME,
     github_repos=[CWL_ICA_GITHUB_REPO],
     github_role_name=f"{CDK_APP_NAME}-prod-umccr-pipelines-deployment-role",
-    env=cdk.Environment(
-        account="472057503814",
-        region="ap-southeast-2"
-    ),
-)
-
-# V2 (single token)
-IcaCredentialsDeployment(
-    app,
-    f"{CDK_APP_NAME}-dev-v2",
-    None,  # Token does not require project context in v2
-    None,  # Token does not require additional project list in v2
-    ICAV2_BASE_URL,
-    SLACK_HOST_SSM_NAME,
-    SLACK_WEBHOOK_SSM_NAME,
-    env=cdk.Environment(
-        account=os.environ["CDK_DEFAULT_ACCOUNT"],
-        region=os.environ["CDK_DEFAULT_REGION"],
-    ),
```

Comment:
I took out the ability for these to be deployed account agnostic - because I don't know the use case for that?

Reply:
Happy to drop this